app/models/invenio_rdm_record_converter.rb

```diff
@@ -293,6 +293,8 @@ def subjects_for_scheme(terms, scheme)
         {id: pid}
       elsif scheme == :subject_name || scheme == :tag
         {subject: term}
+      else
+        puts "------\nUnable to map subject\nFile Id: #{@generic_file.id} Term: #{term} Scheme: #{scheme}\n------"
```

Comment:
This seemed like the easiest way to do this without adding yet another file write out to this, but if we would prefer to write out to a file that would be an easy change to make.

Reply:
we can always redirect the stdout to file anyway, so that's ok.

Reply:
Oh, good point.